modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,171 @@
+
+* This update adds support for {tekton-chains} to get the `imgPullSecret` parameter from the pod template. This update helps you to configure repository authentication based on each pipeline run or task run without modifying the service account.
+
+[id="tekton-hub-new-features-1-9_{context}"]
+=== {tekton-hub}
+
+:FeatureName: {tekton-hub}
+include::snippets/technology-preview.adoc[]
+
+* With this update, you can use an external database, such as Crunchy PostgreSQL with {tekton-hub}, instead of using the default {tekton-hub} database deployed by the Operator. This update helps you to avoid operational management of default databases.
+
+* With this update, you can customize the configuration of {tekton-hub} using the existing cluster configurations settings, instead of relying on external Git repositories for updating the configurations.
+
+
+[id="pac-new-features-1-9_{context}"]
+=== {pac}
+
+* This update supports concurrency limit in the repository CRD to define the maximum number of `PipelineRuns` for a repository at a time. This update also enhances the `PipelineRun` concurrency to execute pipelines in alphabetical order instead of random.
+
+* This update adds a new command `tkn pac logs` for showing the logs of the latest pipeline run for a repository.
+
+* Before this update, the {pac} controller required to restart to take configuration changes into account. With this update, the {pac} controller takes changes into account based on the `pipelines-as-code` configmap.
+
+* This update supports advanced event matching on titles for pull requests and merge requests to GitHub and GitLab. For example, you can run a pipeline if it has the title of the pull request starting with specific words.
+
+* This update adds a new driver to support the Gitea software package.
+
+* With this update, you can reference a remote pipeline in the `pipelineRef:` object using annotations.
+
+* With this update, you can auto-configure new GitHub repositories with {pac}, which sets up a namespace and creates a custom resource for your repository.
+
+* This update provides metrics for pipeline runs with provider information in the user namespace.
+
+* This update provides the following enhancements for the `tkn-pac` plugin:
+** Detects running pipelines correctly 
+** Fixes showing duration when there is no failure completion time
+** Shows an error snippet and highlights the error regular expression pattern in the `tkn-pac describe` command
+** Adds the `use-real-time` switch to the `tkn-pac ls` and `tkn-pac describe` commands
+** Imports the `tkn-pac` logs documentation
+** Shows `pipelineruntimeout` as a failure in the `tkn-pac ls` and `tkn-pac describe` commands.
+** Show a specific pipeline run failure with the `--target-pipelinerun` option.
+      
+* With this update, you can view the errors for your pipeline run in the form of a VS code comment or a small snippet in the GitHub checks. 
```

Comment:
VS code comment?

Reply:
:thinking: 

the release note is like below

```
If there are any errors for your pipelinerun you can now see a display of those errors and addition of a small snippet into the github checks or as vcs comment
```